stock_request_picking_type/tests/test_stock_request.py

```diff
@@ -185,4 +190,4 @@ def test_create(self):
         order = self.request_order.with_user(self.stock_request_user).create(order_vals)
 
         # test create()
-        self.assertEqual(order.picking_type_id, new_pick_type)
+        self.assertNotEqual(order.picking_type_id, new_pick_type)
```

Comment:
Why is this changed ?

Reply:
I remove the test because is not impacts  in the total coverage at the module